local-dev/local-postgres-init.sql

```diff
@@ -0,0 +1,4 @@
+CREATE DATABASE testdb;
+CREATE ROLE dbuser WITH LOGIN ENCRYPTED PASSWORD 'dbpwd';
+CREATE DATABASE janitor;
+CREATE ROLE janitoruser WITH LOGIN ENCRYPTED PASSWORD 'janitorpwd';
```

Comment:
is this file used yet? I don't think we're running postgres locally with skaffold? Next iteration for local db instead of cloud sql?

Reply:
right, we don't need this yet. Will make a separate PR when we need local db